src/test/java/org/springframework/social/bitbucket/api/impl/GroupPrivilegesTemplateTest.java

```diff
@@ -87,7 +87,7 @@ public void testGetPrivilegedGroupsForRepository() throws Exception {
         assertNotNull(firstPrivilegeGroup.getGroup().getMembers());
         assertTrue(firstPrivilegeGroup.getGroup().getMembers().isEmpty());
         assertEquals("Contractors", firstPrivilegeGroup.getGroup().getName());
-        assertTrue(firstPrivilegeGroup.getRepository().getOwner().isTeam());
+        assertTrue(firstPrivilegeGroup.getRepository().getOwner().getIsTeam());
```

Comment:
I think it is ok, when someone operates on original bitbucket API docs. The name of the parameter is "is_team"